### ISSUE TITLE
Assembly модулей

### ISSUE THREAD

**Issue by kiokumicu** [SpectralDragon/ViperMCFlurrySwift#8]
А как быть с Assembly модулей. Я так понимаю при переходе через роутер на другой модуль его assembly не инджектятся ?
А сразу собирать все Assembly при старте приложения мне кажется не очень правильным.

**Comment by oleksandr-zarochintsev** [comment]
@kiokumicu нет, `func assemble(container: Container)` вызывается каждый раз, при открытии модуля
